src/panels/lovelace/components/hui-marquee.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  html,
+  LitElement,
+  PropertyValues,
+  TemplateResult,
+  customElement,
+  css,
+  CSSResult,
+  property,
+} from "lit-element";
+
+const marqueeSpeed = 0.2;
+
+@customElement("hui-marquee")
+class HuiMarquee extends LitElement {
+  @property() public marqueeText?: string;
+  @property() public off?: boolean;
+  private interval?: number;
+  private left: number = 0;
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    if (!this.off) {
```

Comment:
I think that you should check against `changedProperties.has("off")`. Also `off` is a weird property. I would go for `active`

Reply:
hmm we only care if it is active. do we really need to check?